Backend/EmailService.cs

```diff
@@ -24,18 +24,17 @@ public class EmailService : IEmailService
             _environmentSettings = environmentSettings.Value;
         }
 
-        public bool SendFeedbackEmail(string userEmail, string context)
+        public bool SendFeedbackEmail(string userEmail, string userName, string subject, string body)
```

Comment:
What for user name? Sender? Receiver?

Reply:
OK, I checke it once again... It is special method for feedback, so userName is OK... I thought that this method is used also for confirmation